ci/jenkins/test-vm.sh

```diff
@@ -48,6 +43,17 @@ Run K8s e2e community tests (Conformance & Network Policy) or Antrea e2e tests o
         --workdir                Home path for Go, vSphere information and antrea_logs during cluster setup. Default is $WORKDIR.
         --registry               The docker registry to use instead of dockerhub."
 
+# VM configuration
+declare -A LINUX_HOSTS_TO_IP
+declare -A WINDOWS_HOSTS_TO_IP
+declare -a LIN_HOSTNAMES=("vmbmtest0-1")
```

Comment:
The real hostname should not be hardcoded because the testbed could be deployed by other users in community. It's better to add two parameters and use secret text as the input value.

Reply:
The test is running on a customized images (For Windows -cygwin/ssh is installed).
We don't expect user to run this test on his private testbed.

@wenyingd can you share your thoughts on this?

Reply:
Also this PR is only updating the existing CI to use install script (instead of running these commands on the CI script).

Reply:
I agree that using a parameter is more feasible than hard-code, and it can reduce security risk if not exposing the private resource information.

Reply:
@antoninbas @tnqn Would you share your thoughts?

Reply:
I'm fine with keeping it as is given what @Anandkumar26 explained and the fact that the previous script also hardcoded the host name, so anyway it doesn't have to be in this PR's scope. We could parameterize it later when it's required.